lib/resources/dstu2/appointment.yaml

```diff
@@ -92,9 +97,9 @@ fields:
     note: When creating a new appointment, this must be a Patient participant.
     example: |
       "actor": {
-              "reference": "Patient/3886413",
-              "display": "Patient, Test"
-       }
+        "reference": "Patient/4704007",
+        "display": "PATIENT, TEST"
+      }
```

Comment:
Other examples in this file are wrapped in a set of squiggly brackets

Reply:
Addressed with f54f352a457572e639496cf1f96efa62816cd041